services-js/commissions-app/src/server/email/EmailContent.ts

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs';
+import Handlebars from 'handlebars';
+import mjml2html from 'mjml';
+
+require('dotenv').config();
+require('./handlebars-helpers');
+
+export interface TemplateData {
+  commissionNames: string[];
+  firstName: string;
+  lastName: string;
+  email: string;
+  applicationId: string;
+  uri?: string;
+}
+
+interface Templates {
+  MJML: any;
+  TEXT: any;
+}
+
+interface RenderedEmailBodies {
+  html: string;
+  text: string;
+}
+
+export default class EmailContent {
+  handlebarsTemplates: any;
+
+  constructor() {
+    this.handlebarsTemplates = {
+      applicant: '',
+      policyOffice: '',
+    };
+
+    this.handlebarsTemplates.applicant = this.readTemplateFiles('applicant');
+
+    this.handlebarsTemplates.policyOffice = this.readTemplateFiles(
+      'policyOffice'
+    );
+  }
+
+  public renderApplicantBodies(data: TemplateData): RenderedEmailBodies {
+    return this.renderEmailBodies('applicant', data);
+  }
+
+  public renderPolicyOfficeBodies(data: TemplateData): RenderedEmailBodies {
+    return this.renderEmailBodies(
+      'policyOffice',
+      data,
+      process.env.COMMISSIONS_URI
+    );
+  }
+
+  private renderEmailBodies(
+    recipient: 'applicant' | 'policyOffice',
+    data: TemplateData,
+    uri?: string
+  ): RenderedEmailBodies {
+    const templateData = { ...data, uri };
+
+    const htmlTemplate = Handlebars.compile(
```

Comment:
Ideally you’d `compile` the templates in the `constructor` so that it only has to be done once.